tests/test_app.py

```diff
@@ -107,7 +107,7 @@ def test_add_page_set_route_nested(app: App, index_page):
         index_page: The index page.
     """
     assert app.pages == {}
-    app.add_page(index_page, path="/test/nested")
+    app.add_page(index_page, route="/test/nested")
```

Comment:
Add error handling for this breaking change

Reply:
Added a deprecation warning